_includes/cockroachcloud/cluster-operator-prereq.md

```diff
@@ -0,0 +1 @@
+Either the [Cluster Operator](authorization.html#cluster-operator) or [Cluster Administrator](authorization.html#cluster-administrator) role on a pre-existing Dedicated cluster, or the [Cluster Creator](authorization.html#cluster-creator) role in order to create a new cluster.
```

Comment:
I think we should remove "dedicated" from here coz soon IP allowlisting and PrivateLink are going to be available for that offering too.